client/src/components/ApplicantsList.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { Button } from "semantic-ui-react";
+import { getAllProfiles } from "../api/profiles";
+
+const ApplicantsList = () => {
+  const [profiles, setProfiles] = useState([]);
+  const fetchProfiles = async () => {
+    const response = await getAllProfiles();
+    setProfiles(response);
+  };
+
+  useEffect(() => {
+    fetchProfiles();
+  }, []);
+
+  return profiles.map(profile => {
+    return (
+      <div class="ui two column grid raised segment">
```

Comment:
Since we are using the React Semantic UI component library in this project, we should use them when possible. It looks like we're using a grid here so let's change it to use the `<Grid>` and `<Grid.Column>` components as demonstrated in https://react.semantic-ui.com/collections/grid/#variations-column-width. What we find is that we no longer need to add (as much) CSS classes since they are encapsulated within the React Semantic UI components!

Reply:
the react semantic ui exists in two formats, it can be either imported as Components or used as classes. There are no additional CSS classes added in the codes. However, I will be changing all the semantic ui classes to Components to synchronise across the pages.